main/lsp/LSPTypechecker.cc

```diff
@@ -718,6 +718,23 @@ void LSPTypechecker::setSlowPathBlocked(bool blocked) {
     slowPathBlocked = blocked;
 }
 
+void LSPTypechecker::updateGsFromOptions(const DidChangeConfigurationParams &options) const {
+    this->gs->trackUntyped = options.settings->highlightUntyped.value_or(this->gs->trackUntyped);
+
+    if (options.settings->enableTypecheckInfo.has_value() ||
+        options.settings->enableTypedFalseCompletionNudges.has_value() ||
+        options.settings->supportsOperationNotifications.has_value() ||
+        options.settings->supportsSorbetURIs.has_value()) {
+        auto msg =
+            "Currently `highlightUntyped` is the only updateable setting using the workspace/didChangeConfiguration "
+            "notification";
+        config->logger->error(msg);
```

Comment:
```suggestion
```

If you're going to send it back via showMessage, you don't also need to send it with the logger, as all `config->output->write` calls log what they write:

https://github.com/sorbet/sorbet/blob/master/main/lsp/LSPOutput.cc#L59